programs/bpf_loader/src/syscalls/mem_ops.rs

```diff
@@ -413,16 +437,37 @@ impl<'a> MemoryChunkIterator<'a> {
             len,
             "unknown",
         ))?;
+
         Ok(MemoryChunkIterator {
             memory_mapping,
+            accounts,
             access_type,
             initial_vm_addr: vm_addr,
             len,
             vm_addr_start: vm_addr,
             vm_addr_end,
+            account_index: 0,
+            is_account: None,
         })
     }
 
+    fn is_account(&mut self, region: &MemoryRegion) -> bool {
```

Comment:
Somewhat confusing that a `is_x` method is `&mut self` not `&self`.

Reply:
Yes, I agree. It is better inlined (done now)